applications/rag/README.md

```diff
@@ -41,6 +51,11 @@ gcloud container node-pools create g2-standard-24 --cluster <cluster-name> \
  --node-locations $REGION-a,$REGION-b --location=$REGION
 ```
 
+3. Ensure your k8s client is using the correct cluster by running:
```

Comment:
i think this is only needed for the verify setup step, should we move it down?

Reply:
Done.